src/any.rs

```diff
@@ -5,7 +5,7 @@ use crate::parser::{ImmutableString, INT};
 use crate::r#unsafe::{unsafe_cast_box, unsafe_try_cast};
 
 #[cfg(not(feature = "no_float"))]
-use crate::parser::FLOAT;
```

Comment:
Better keep `crate::parser::FLOAT` because that frees us in the future to remap anything under the `crate::` root.

Reply:
I changed this because it gave me a compiler error before. Now it doesn't, so I will fix it (and the two other instances as well).